tests/Domain/Package/PackageManagerTest.php

```diff
@@ -142,9 +160,18 @@ public function testConstructionAndGetters(): void {
       $actual_package_list[$name] = 0;
     }
 
+    // Normalize expected dependency list for clearer comparison.
+    $actual_dependency_list = [];
+    foreach (array_keys($all_dependencies) as $name) {
+      $actual_dependency_list[$name] = 0;
+    }
+
     self::assertEquals(self::EXPECTED_PACKAGE_LIST, $actual_package_list, 'Set/got all packages.');
     self::assertInstanceOf(Package::class, reset($all_packages), 'Got packages as Package objects.');
     self::assertEquals('drupal/module2', $package->getPackageName(), 'Got package by name.');
+    self::assertEquals(self::EXPECTED_DEPENDENCY_LIST, $actual_dependency_list, 'Set/got all dependencies.');
+    self::assertEquals(TRUE, $package->isCompanyPackage(), 'Set/got all dependencies.');
+    self::assertEquals(FALSE, $manager->get('drupal/dependency1')->isCompanyPackage(), 'Set/got all dependencies.');
```

Comment:
Looks like all three assertions accidentally got the same "Set/got all dependencies" assertion message.

Reply:
oops, let me correct them.